pkg/distro/image_config.go

```diff
@@ -147,6 +147,26 @@ type WSLConfig struct {
 	BootSystemd bool `yaml:"boot_systemd,omitempty"`
 }
 
+type WSLDistributionOOBEConfig struct {
+	DefaultUID  *int   `yaml:"default_uid,omitempty"`
+	DefaultName string `yaml:"default_name,omitempty"`
+}
+
+type WSLDistributionShortcutConfig struct {
+	Enabled bool   `yaml:"enabled,omitempty"`
+	Icon    string `yaml:"icon,omitempty"`
+}
+
+type WSLDistributionConfig struct {
+	OOBE     *WSLDistributionOOBEConfig     `yaml:"oobe,omitempty"`
+	Shortcut *WSLDistributionShortcutConfig `yaml:"shortcut,omitempty"`
+}
+
+type WSL struct {
+	Config             *WSLConfig             `yaml:"config,omitempty"`
+	DistributionConfig *WSLDistributionConfig `yaml:"distribution_config,omitempty"`
+}
```

Comment:
I think that we should be moving these to `pkg/customizations` instead of adding them to the `distro` package. The same applies to the `Sysconfig` struct below, which creates the wrong precedence.

Reply:
By that you mean you would (eventually) want to expose them to (blueprint) users?

Reply:
I understand that the directory name makes this impression. I mean, maybe, but that's not what's my intention. We started with osbuild stage options in the `ImageConfig` and wanted to add abstractions gradually, which were osbuild-agnostic. And definitions in the `pkg/customizations` are the place where we added them. These are not meant to be related to Blueprints, but in general to customizations done to images (whether via BP or in `ImageConfig`). The idea is that BP customizations get translated to these data structures, which then get merged with the `ImageConfig`.

Reply:
Ok, and do we already have the merging machinery?

Reply:
Thanks, then it's clear what I'll change :)